packages/cli-core/package.json

```diff
@@ -19,6 +19,7 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@inquirer/prompts": "^3.0.0",
-    "execa": "^8.0.1"
+    "execa": "^8.0.1",
+    "kleur": "^4.1.5"
```

Comment:
Why do we choose to use kleur instead of other packages? 

It'd prevent us from a lot of headaches if we don't introduce any packages at all https://stackoverflow.com/questions/9781218/how-to-change-node-jss-console-font-color

Reply:
evaluated kleur, picocolors and chalk, kluer has chalk like syntax and is lighter. 
also kluer has zero dependancies
Benchmarks: https://github.com/lukeed/kleur#benchmarks 
we can switch to picocolors: https://github.com/alexeyraspopov/picocolors#benchmarks as well.